### DISCUSSION TITLE
Verbesserungsvorschlag Wort Verbrauchsgut  Polishing

### DISCUSSION THREAD

**Discussion by BTSVDanielk** [rjcncpt/StarCitizen-Deutsch-INI#558] [Übersetzungsvorschläge]
Guten Morgen,

Beim Wort Verbrauchsgut würde ich besser finden, wenn man nach dem s ein Bindestrich oder Minus hinsetzt -
Damit das g vielleicht in die zweite Zeile rutscht. Vielleicht geht es auch anders das, dass g in die zweite Zeile rutscht? Das würde besser aussehen, wenn die Silben getrennt würden.
Ich weiß aber auch in diesen Fall nicht, was im Englischen steht. In das Menü kommt man, wenn man ein Medpen in
der Hand hat und alt+F drückt.

![ScreenShot-2024-07-02_09-46-56-8B0](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/c4768269-f24d-45a7-bc94-5c2198a9b328)

**Comment by rjcncpt** [comment]
Danke dir @BTSVDanielk 
Ich mach einfach Gegenstand wegstecken :)
